runsc/config/flags.go

```diff
@@ -101,6 +101,7 @@ func RegisterFlags(flagSet *flag.FlagSet) {
 	flagSet.Int("dcache", -1, "Set the global dentry cache size. This acts as a coarse-grained control on the number of host FDs simultaneously open by the sentry. If negative, per-mount caches are used.")
 	flagSet.Bool("iouring", false, "TEST ONLY; Enables io_uring syscalls in the sentry. Support is experimental and very limited.")
 	flagSet.Bool("directfs", false, "directly access the container filesystems from the sentry. Sentry runs with higher privileges.")
+	flagSet.Bool("enable-cuda", false, "TEST ONLY; Enables CUDA support. Not yet implemented.")
```

Comment:
IMO we can name the flag `cuda` for consistency with other flags. We have a lot of flags that enable stuff, but they are all just named with the feature they enable.